packages/source-drupal/src/entities/TaxonomyTerms.js

```diff
@@ -0,0 +1,71 @@
+const Entity = require('./Entity')
+
+class TaxonomyTypes extends Entity {
+  constructor(source, type, url) {
+    // config == { entityType, entityName, type, url }
+    super(source, type, url)
+  }
+
+  async initialize() {
+    try {
+      console.log('Intializing TaxonomyTypes...')
+
+      await this.fetchData()
+      await this.buildGraphQl()
+
+      console.log('TaxonomyTypes initialzed...')
+    } catch ({ message }) {
+      throw new Error(message)
+    }
+  }
+
+  async buildGraphQl() {
+    let { store } = this.source
+    let { addContentType } = store
+    
+    const config = this.createGraphQlType()
+    this.graphQlContentType = addContentType(config)
+
+    for (let item of this.response) {
+      let {
+        id,
+        attributes: {
+          name,
+          created,
+          description: {
```

Comment:
Can throw errors if `description` is null, as in this JSON:

```
{
  "type": "categories",
  "id": "1f808af2-bd6d-4576-a1df-5a74eb343545",
  "attributes": {
    "internalId": 31,
    "status": true,
    "name": "Accompaniment",
    "description": null,
    "weight": 0,
    ...
  }
}
```

Reply:
Doh! I hate how Javascript handles `nulls` in destructuring. It gets me every time. I wish it would see it as a `falsey` value and fallback to the default value that is provided. Ugh.